src/parser/ad_parser.js

```diff
@@ -27,29 +28,26 @@ export function parseAd(adElement) {
 
     parserUtils.copyNodeAttribute('id', adElement, adTypeElement);
     parserUtils.copyNodeAttribute('sequence', adElement, adTypeElement);
-
-    if (adTypeElement.nodeName === 'Wrapper') {
-      return parseWrapper(adTypeElement);
-    } else if (adTypeElement.nodeName === 'InLine') {
-      return parseInLine(adTypeElement);
-    }
+    return parseAdElement(adTypeElement, emit);
```

Comment:
I'm not sure what we gained by refactoring this part. Previously parsing inlines and wrappers had a more separate logic, and now parseAdElement does everything. To me it was clearer how inlines and wrappers were parsed previously. Let's discuss this :)